packages/client/tests/functional/issues/21552-datetime-representation-compatibility/tests.ts

```diff
@@ -0,0 +1,54 @@
+import execa from 'execa'
+import fs from 'fs/promises'
+import path from 'path'
+
+import { DatasourceInfo } from '../../_utils/setupTestSuiteEnv'
+import testMatrix from './_matrix'
+// @ts-ignore
+import type { PrismaClient } from './node_modules/@prisma/client'
+
+declare const prisma: PrismaClient
+declare const datasourceInfo: DatasourceInfo
+
+/**
+ * Regression test for https://github.com/prisma/prisma/issues/21552
+ */
+testMatrix.setupTestSuite(
+  (_suiteConfig, suiteMeta) => {
+    test('can read back DateTime created via native connector', async () => {
+      // Copy the seed script to the generated directory so that it imports the
+      // correct client, and so that the schema-relative sqlite database path is
+      // resolved the same way as in the test.
+      await fs.cp(path.join(__dirname, '_seed.ts'), path.join(suiteMeta.generatedFolder, '_seed.ts'))
+
+      // Use a separate process for seeding to use native drivers. We can't turn
+      // the native quaint executor back on in the current process.
```

Comment:
Do you mean that you need this because `PRISMA_DISABLE_QUAINT_EXECUTORS` may be disabled when running against driver adapters?

Reply:
Yes, and in fact that's the main thing that we are testing, the test is not even really relevant without driver adapters.

Reply:
I expanded the comment a tiny bit so it's hopefully more clear without the context.